### ISSUE TITLE
Losses in training have a lot of shock

### ISSUE THREAD

**Issue by walkingmelon** [JiaRenChang/PSMNet#104]
Dear@JiaRenChang ，When I used PSMNet to train SceneFlow datasets, the loss was quite volatile. Is the data set not clean? And how do I deal with it? Should I delete some unreliable data? Look forward to your reply！

**Comment by walkingmelon** [comment]
Thank you for your timely reply. I also noticed the restrictions on disparity in the code, and I didn't modify the code. Attached picture is my training loss.
![1773168584](https://user-images.githubusercontent.com/41433319/50396103-a2dd0180-07a2-11e9-89e8-bce4fa20405d.jpg)

**Comment by JiaRenChang** [comment]
@walkingmelon 
Yep, it is fluctuating.
Which pass of Scene Flow do you use ?
We use the cleanpass version.

**Comment by walkingmelon** [comment]
Yes, I use the same data set. Maybe my data set is contaminated. I will now download the dataset again. I hope I will have good results.
